src/External/Libraries/Bridge/Laravel/Providers/EoneoPayApiSdkServiceProvider.php

```diff
@@ -41,5 +44,27 @@ public function register(): void
                 new ExceptionFactory()
             );
         });
+
+        // V2 client
+        $this->app->singleton('eoneopay_v2_api_client', static function () {
```

Comment:
Instead of defining a new http client we can instead have a version const on the objects themselves - then the main sdk service can just build the header for each object it receives